docs/approvals/jira-service-management/index.md

```diff
@@ -0,0 +1,211 @@
+license register for the [JSM Early Access Program](https://octopusdeploy.
+typeform.com/servicenow-eap)
+:::
+
+## Overview
+
+The Octopus Deploy/JSM integration allows users to block the execution of 
+specifically configured deployments unless they have a corresponding approved JSM  **Change 
+Request** (aka issue).
+
+To enable this behavior, both the Octopus Project and Environment you are deploying to must be 
+configured and the JSM configuration is set up before deployments can be managed.
+
+| Project | Environment | Outcome |
+|--|--|--|
+| Change controlled| Change controlled| Approval required for deployment |
+| **_Not_** Change controlled| Change controlled| No approval required |
+| Change controlled| **_Not_** Change controlled| No approval required |
+
+## Getting started
+
+The JSM integration requires Octopus **2022.3.12101** or later and an Octopus license with the
+JSM Integration feature enabled.
+
+Before you can use the Octopus Deploy/JSM integration, you'll need to:
+
+1. Create a service account in JSM for use by Octopus
+1. Request and install a new Octopus license required to enable the JSM feature.
+1. Configure a connection from Octopus to JSM.
+1. Configure which deployments require an approved CR.
+
+### Configuring Jira Service Management
+
+:::hint
+The instructions in this section will require a JSM Administrator.
+:::
+
+The Octopus Deploy / JSM integration requires security configuration in your target JSM instance.
+
+The integration will require a user account in JSM. The recommendation is to create a 
+service account specifically for Octopus.
+
+Take note of the password assigned or generated for this user.
+
+### Licensing
+
+For the JSM approval checks to be performed as part of the deployment process, an appropriate Octopus license must be configured in your Octopus instance.
+
+A JSM enabled Octopus license must be requested from Octopus directly, and cannot be managed through the self-service process. To request a license register for the [JSM Early Access Program](https://octopusdeploy.typeform.com/servicenow-eap)
+
+Once you have received your feature-enabled license, you can install it by navigating to **{{Configuration, License}}**.
+
+An enabled license will include a block similar to below:
+
+```xml
+<Features>
+  <JiraServiceManagement>
+    <ValidTo>...</ValidTo>
+  </JiraServiceManagement>
+</Features>
+```
+
+### Configuring JSM connections
+
+:::hint
+The instructions in this section will require an Octopus Deploy Manager or Administrator
+:::
+
+To connect your Octopus Deploy instance to JSM, navigate to **Configuration ➜ Settings ➜ 
+Jira Service Management Integration**.
+
+Check the **Enabled** option
+![JSM Integration Settings page](images/jsm-connections-1.png "width=500")
+
+Click on **ADD CONNECTION** and fill out the details.
+The JSM Base Url should be the root URL and include the protocol e.g. `https://`
```

Comment:
nit: line break

Reply:
not sure what I'm missing - 87 should end on "https://" - then 89 is the next instruction